disnake/http.py

```diff
@@ -101,6 +102,16 @@ def _workaround_set_api_version(version: Literal[9, 10]) -> None:
     Route.BASE = f"https://discord.com/api/v{_API_VERSION}"
 
 
+def _get_user_agent() -> str:
+    user_agent = (
+        "DiscordBot (https://github.com/DisnakeDev/disnake {0}) Python/{1[0]}.{1[1]} aiohttp/{2}"
+    )
+    return user_agent.format(__version__, sys.version_info, aiohttp.__version__)
+
+
+USER_AGENT: Final[str] = _get_user_agent()
```

Comment:
Is there a reason for this to be a separate method at all, if all the inputs are constant? Versions aren't going to change over the lifetime of the process

Reply:
https://github.com/DisnakeDev/disnake/pull/959/commits/c6d53c31828d56018098f69724b72ed6a66c8be6